src/Faction/ui/ShareOption.tsx

```diff
@@ -55,9 +55,10 @@ export function ShareOption({ rerender }: { rerender: () => void }): React.React
   return (
     <Paper sx={{ my: 1, p: 1 }}>
       <Typography>
-        You can share free RAM of your home computer with your faction to get a bonus multiplier for reputation gain.
-        Each time you share your free RAM, you get a boost for {ShareBonusTime / 1000} seconds. You can share free RAM
-        of other servers that you have admin rights by using ns.share() API.
+        You can share free RAM of your home computer with your factions to get a bonus multiplier for reputation gain.
+        Each time you share your free RAM, you get a boost for {ShareBonusTime / 1000} seconds. After that, you lose the
+        boost and get back your shared RAM. You can share free RAM of other servers that you have admin rights by using
+        ns.share() API.
```

Comment:
```suggestion
        boost and get back your shared RAM. You can share free RAM of other servers that you have admin rights on by using
        the ns.share() API.
```
or "that you have NUKE'd"